OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs

```diff
@@ -269,7 +269,6 @@ void RemoveCamera(Actor camera)
 
 			camera.QueueActivity(new Wait(info.CameraRemoveDelay));
 			camera.QueueActivity(new RemoveSelf());
-			camera = null;
```

Comment:
Why did you remove this? It prevents `RemoveCamera` from queuing activities twice. Although I guess that shouldn't be a problem any more. If it is not a problem, is it possible to remove `beacon = null;` below too?

Reply:
`camera` here is a parameter, so assigning null to it wouldn't be visible outside this method anyway. Removing the `beacon = null` looks fine. The latter won't have been detected by the rules since it's inside a callback, but it looks safe to remove nonetheless.

Reply:
I also found another occurrence in `AirstrikePower` where I removed it as well.